packages/ui/src/components/rich-markdown-editor.tsx

```diff
@@ -217,7 +216,7 @@ export function RichMarkdownEditor({
           <div>Uploading image...</div>
         </div>
       ) : null}
-      <input className="hidden" onChange={handleFileChange} ref={fileInputRef} type="file" />
+      <input className="hidden" ref={fileInputRef} type="file" />
```

Comment:
`onChange` is an optional prop, providing an empty function does nothing, right?